codersdk/agentsdk/convert.go

```diff
@@ -268,6 +280,19 @@ func ProtoFromApp(a codersdk.WorkspaceApp) (*proto.WorkspaceApp, error) {
 	if !ok {
 		return nil, xerrors.Errorf("unknown health %s", a.Health)
 	}
+
+	var openIn proto.WorkspaceApp_OpenIn
+	switch a.OpenIn {
+	case "slim-window":
```

Comment:
Use the constants (i.e. `codersdk.WorkspaceAppOpenInSlimWindow`) instead.

Reply:
Removed as this piece of code is not relevant anymore.